src/http-gateways/trustless-gateway.md

```diff
@@ -235,6 +253,21 @@ In case both are present in the request, the value from the [`Accept`](#accept-r
 
 Below MUST be implemented **in addition** to "HTTP Response" of :cite[path-gateway].
 
+## Status codes
```

Comment:
We should refer to https://specs.ipfs.tech/http-gateways/path-gateway/#response-status-codes and avoid duplicating code list. 

Follow convention from https://specs.ipfs.tech/http-gateways/trustless-gateway/#content-location-response-header and https://specs.ipfs.tech/http-gateways/trustless-gateway/#http-response where we refer to path-gateway spec where possible.